src/components/PersonLink/PersonLink.tsx

```diff
@@ -0,0 +1,52 @@
+import { Link, useParams } from 'react-router-dom';
+import { Person } from '../../types';
+import classNames from 'classnames';
+
+type Props = {
+  person: Person;
+};
+export const PersonLink: React.FC<Props> = ({ person }) => {
+  const { slug } = useParams();
+  const { name, sex, born, died, fatherName, motherName, mother, father } =
+    person;
+  const female = sex === 'f';
+  const notParents = '-';
```

Comment:
This constants should be created outside the component and be named in `UPPER_CASE`